services/thermal_mgr/thermal_mgr.c

```
@@ -43,18 +43,39 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
-
+  if(!event){
+    ERR_CODE_INVALID_ARG;
```

should return this err code

---

oops